docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -21,26 +21,26 @@ NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image
 [id='examine_logs_{context}']
 == Examining and collecting {WebName} logs
 
-Each Pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. The following steps discuss viewing and obtaining these logs.
+Each pod is configured to provide detailed logging that assists with narrowing down the precise cause of an issue. You can use the OpenShift Container Platform console or the CLI to download pod logs.
 
-.OpenShift console
+.OpenShift Container Platform console
 
-You can download a Pod's log with the OpenShift Container Platform console.
+You can download a pod log with the OpenShift Container Platform console.
 
 .Procedure
 
-. Open the OpenShift console and navigate to *Applications* -> *Pods*.
-. Click the name of the Pod that you wish to examine. To examine the current {WebName} Pod, select the `mta-web-console-<POD_NAME>` that is in a `Running` state.
+. Open the OpenShift Container Platform console and navigate to *Applications* -> *Pods*.
```

Comment:
When referring to the OCP console, "OpenShift web console" is acceptable. We need "OpenShift" to distinguish this console from the MTA console. You do not need to put in the full name each time if the first mention of OCP in module is full. Please review the branding guidelines. https://docs.google.com/spreadsheets/u/2/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/pubhtml?gid=1259317633&single=true